src/liftdrag_plugin/liftdrag_plugin.cpp

```diff
@@ -258,9 +311,9 @@ void LiftDragPlugin::OnUpdate()
   // if forwardI is in the same direction as lift, alpha is positive.
   // liftI is in the same direction as forwardI?
   if (liftI.Dot(forwardI) >= 0.0)
```

Comment:
Remove the if/else structure if it isn't being used.

Reply:
it is being used

Reply:
Ah ok I see - sorry about that. Do you know why lift would ever have a component in the opposite direction of forward? I don't quite understand that

Reply:
I created an issue to later look into validity of this logic when liftI and upwardI are not perpendicular